ses/index.cgi

```diff
@@ -13,7 +13,7 @@ use warnings;
 use lib qw(.. ../lib ../local/lib/perl5);
 
 use Bugzilla ();
-use Bugzilla::Logging;
+use Bugzilla::Logging qw( FATAL WARN );
```

Comment:
This is not correct. Use the bare import for this.
I'll add a warning about this later.